central/scannerdefinitions/handler/updater_test.go

```diff
@@ -23,9 +23,7 @@ const (
 
 	mappingURL = "https://storage.googleapis.com/definitions.stackrox.io/redhat-repository-mappings/mapping.zip"
 
-	cvssURL = "https://storage.googleapis.com/scanner-v4-test/nvd-bundle/nvd-data.tar.gz"
-
-	v4VulnURL = "https://storage.googleapis.com/scanner-v4-test/vulnerability-bundles/dev/vulns.json.zst"
+	v4VulnURL = "https://storage.googleapis.com/definitions.stackrox.io/vulnerability-bundles/dev/vulns.json.zst"
```

Comment:
We want to use https://definitions.stackrox.io/, right? I am thinking that the user's firewall/policies/etc would reference that instead.

Reply:
make sense, I will update that